### ISSUE TITLE
how to change the number of class ?

### ISSUE THREAD

**Issue by 18242360613** [qijiezhao/pseudo-3d-pytorch#1]
Thank you for you great work , I want to use it in my work ,but I can't change the number of the class. Could please tell how to change the number of the class and how to finetune on my own dataset

**Comment by qijiezhao** [comment]
Sorry to reply lately, I finetuned p3d in my own works. Here I show some of my examples:

     from p3d_model import *
     net=P3D131(modality='RGB', dropout=0.5)
     num_class=101
     net=transfer_model(net,num_classes=num_class)
     
     #here is the definition of transfer_model:
     def transfer_model(model,num_classes=21):
         num_ftrs=model.fc.in_features
         model.fc=nn.Linear(num_ftrs,num_classes)
         return model
